rpcserver.go

```diff
@@ -6141,15 +6140,14 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 		},
 		GetAlias:   r.server.aliasMgr.GetPeerAlias,
 		BestHeight: r.server.cc.BestBlockTracker.BestHeight,
-		QueryBlindedRoutes: func(amt lnwire.MilliSatoshi) (
-			[]*route.Route, error) {
-
-			return r.server.chanRouter.FindBlindedPaths(
-				r.selfNode, amt,
-				r.server.defaultMC.GetProbability,
+		QueryBlindedPaths: func() ([][]blindedpath.BlindedHop, error) {
+			return routing.FindBlindedPaths(
+				r.server.graphDB,
+				r.selfNode,
 				blindingRestrictions,
 			)
 		},
+		ProbabilitySource: r.server.defaultMC.GetProbability,
```

Comment:
I think we can remove this one as well and do this prior to entering the invoice package.